magicbyirineuTests/Modules/CardList/MagicFlowLayoutSpec.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  MagicFlowLayoutSpec.swift
+//  magicbyirineuTests
+//
+//  Created by daniel.da.cunha.lima on 28/02/2019.
+//  Copyright © 2019 DanielLima. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Nimble_Snapshots
+import KIF
+
+@testable import magicbyirineu
+
+class MagicFlowLayoutSpec:QuickSpec{
+    
+    override func spec() {
+        
+        var sut:MagicFlowLayoutMock!
+        var viewController:CollectionViewControllerMock!
+        
+        beforeEach {
+            
+            sut = MagicFlowLayoutMock()
+            
+            let view = CollectionMock(mockLayout: sut)
+            viewController = CollectionViewControllerMock(mockView: view)
+            
+            UIApplication.shared.keyWindow?.rootViewController = viewController
+            
+            _ = viewController.view
+            
+        }
+        
+        
+        context("when it is initialized"){
+            
+            it("minimumInteritemSpacing should be 0"){
```

Comment:
```suggestion
            it("has minimumInteritemSpacing equal to 0"){
```

Reply:
Da uma olhada nos `it`. `has` faz mto mais sentido quando se trata de teste pois should dá uma ideia de que deveria mas n está e o `has` da uma sensação de que realmente tem q ter aquilo

Reply:
Resolvido, de fato faz mais sentido usar has. Vlw!